plotter.go

```diff
@@ -70,6 +71,20 @@ func Plot(expr PlotExpr, queryTime time.Time, duration, resolution time.Duration
 		resolution,
 	)
 	if err != nil {
+		_ = bugsnag.Notify(errors.Wrap(err, "error querying Prometheus"), nil,
```

Comment:
not sure if we need to wrap this error before notifying

Reply:
I wondered if it might make it a little easier to pinpoint, we also submit wrapped errors elsewhere, but i'll remove it for now.

Reply:
it should include the file and line number in the stacktrace